test/fixtures/full-import-specifier/expected.js

```diff
@@ -49,3 +49,11 @@ Number(a);
 typeof a === "bigint";
 BigInt.asIntN(64, 42n);
 BigInt.asUintN(64, 42n);
+const buffer = new ArrayBuffer(16); // 0x7FFFFFFFFFFFFFFFn, the highest possible BigInt value that fits in
```

Comment:
It's weird that the line break before the comment is removed, but that seems unrelated to this patch.

Reply:
Yeah, Babel’s pretty-printing does not try to match the source format. It doesn’t matter too much though since this probably gets minified in practice afterwards anyways.